engines/sci/detection.cpp

```diff
@@ -119,10 +119,10 @@ static const PlainGameDescriptor s_sciGameTitles[] = {
 	{"hoyle4",          "Hoyle Classic Card Games"},
 	{"inndemo",         "ImagiNation Network (INN) Demo"},
 	{"kq6",             "King's Quest VI: Heir Today, Gone Tomorrow"},
-	{"laurabow2",       "Laura Bow 2: The Dagger of Amon Ra"},
-	{"qfg1vga",         "Quest for Glory I: So You Want to Be a Hero"},	// Note: There was also a SCI0 version of this (further up)
+	{"laurabow2",       "Roberta Williams' Laura Bow in The Dagger of Amon Ra"}, //also called Laura bow II
+	{"qfg1vga",         "Quest for Glory: So You Want to Be a Hero"},	// Note: There was also a SCI0 version of this (further up)
 	{"qfg3",            "Quest for Glory III: Wages of War"},
-	{"sq5",             "Space Quest V: The Next Mutation"},
+	{"sq5",             "Space Quest V: Roger Wilco - The Next Mutation"},
```

Comment:
Here's where things get problematic, as SQ5 is referred to as "Space Quest V: The Next Mutation" in rereleased as well as the hint book that came out at the time

Reply:
So I am thinking if the release is on a different engine then it will be changed there; however, if it is all on the same engine then this title is the one on the original box cover and it makes sense to try and keep to that standard through out. I should leave a note next to it explaining this issue though(with proper capitalization). I am of course welcome if there is a better idea.